app/common/elasticsearchclient.py

```diff
@@ -240,19 +240,25 @@ def best_hit_search(self,
         # current_app.logger.debug("Got %d Hits in %ims" % (res['hits']['total'], res['took']))
         data = []
         
-        for res in results['responses']:
-            res_data = [];
-            for hit in res['hits']['hits']:
```

Comment:
below we know that es multiseaarch is going to return as many responces as we had queries. If no hits returned we still generate datapoint to match. We do not need to pass back query (which in our case is the targetName, because we do know that order is preserved, but doing it for convenience)
The way to determine fuzzy search: lowercase both  and compare q=data['approved_symbol']. If they are not the same, then it is a fuzzy search.